.vale/styles/RedHat/Spelling.yml

```diff
@@ -212,9 +214,11 @@ filters:
   - Endian
   - endif
   - Esprima
+  - Etcd
```

Comment:
```suggestion
  - etcd
```

for consistency with:

VALE: Use 'etcd' rather than 'Etcd'.

Reply:
Updated.